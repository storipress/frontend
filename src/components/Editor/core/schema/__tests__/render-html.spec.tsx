import { h } from '../render-html'

describe('h', () => {
  it('render array children', () => {
    const res = h('div', {}, [null, h('span', {}, 'foo')] as any)

    expect(res).toMatchSnapshot()
  })

  it('render normal children', () => {
    const res = h('div', {}, h('span', {}, 'foo'))

    expect(res).toMatchSnapshot()
  })

  it('render with null children', () => {
    const res = h('div', {}, null as any, h('span', {}, 'foo'))

    expect(res).toMatchSnapshot()
  })
})
