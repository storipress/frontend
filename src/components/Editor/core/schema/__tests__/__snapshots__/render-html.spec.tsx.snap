// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`h render array children 1`] = `
Array [
  "div",
  Object {},
  Array [
    "span",
    Object {},
    "foo",
  ],
]
`;

exports[`h render normal children 1`] = `
Array [
  "div",
  Object {},
  Array [
    "span",
    Object {},
    "foo",
  ],
]
`;

exports[`h render with null children 1`] = `
Array [
  "div",
  Object {},
  Array [
    "span",
    Object {},
    "foo",
  ],
]
`;
